modules/ttdBidAdapter.js

```diff
@@ -387,6 +382,10 @@ export const spec = {
       ext: getExt(firstPartyData)
     }
 
+    if (firstPartyData && firstPartyData.bcat) {
+      topLevel.bcat = firstPartyData.bcat;
```

Comment:
can you document if this overwrites or appends existing settings for an account on the backend? I think append is not particularly workable, and overwrite is preferred.

Why not battr and badomain as well?

Reply:
Sorry, badv

Reply:
We would also like to see support for battr and badv here.

Reply:
I'll add battr/badv. We're still discussing behavior internally.

Reply:
thanks!

Reply:
We decided to make the blocking additive (docs will reflect this).

Reply:
Can we discuss? This decision breaks all utility of this feature

Reply:
@julian-burger-ttd can you confirm the badv battr changes above are planned for this pr?

Reply:
Confirming for this PR. Haven't had a chance to get to it yet, though. As for overwrite vs. additive we having some internal discussion as there are multiple use cases to consider.

Reply:
Updated the code. Docs updates (separate MR anyway) will follow.